internal/apps/dop/endpoints/pipeline.go

```diff
@@ -394,26 +394,11 @@ func (e *Endpoints) pipelineRun(ctx context.Context, r *http.Request, vars map[s
 		ConfigManageNamespaces: []string{utils.MakeUserOrgPipelineCmsNs(identityInfo.UserID, p.OrgID)},
 		Secrets:                utils.GetGittarSecrets(p.ClusterName, p.Branch, p.CommitDetail),
 	}); err != nil {
-		var apiError, ok = err.(*errorresp.APIError)
-		if !ok {
-			// Failed to convert to apiError type, return the error passed by the pipeline component
-			return errorresp.ErrResp(err)
-		}
-
-		ctxMap, ok := apiError.Ctx().(map[string]interface{})
-		if !ok {
-			// Interface converted to map[string]interface{} fails and returns the error passed by the pipeline component
-			return errorresp.ErrResp(err)
+		runningPipelineLink, ok := e.ProjectPipelineSvc.TryGenRunningPipelineLinkFromErr(p.PipelineDTO.OrgName, p.PipelineDTO.ProjectID, p.PipelineDTO.ApplicationID, err)
+		if ok {
+			return errorresp.ErrResp(apierrors.ErrParallelRunPipeline.InvalidState(fmt.Sprintf("failed to run pipeline, there is already running: %s", runningPipelineLink)))
```

Comment:
adjust your error msg, it's a little bit strange near `there is already running`

Reply:
change to `already running link:`
![image](https://user-images.githubusercontent.com/30427474/177294688-d487b533-1a59-4ef6-ad79-0a97404ad640.png)